File: data_types_project/cpp/records.cpp
#include <iostream>
#include <string>
using namespace std;

struct Student {
    string name;
    int id;
};

int main() {
    Student student = {"Bryce", 1001};

    cout << "Student Name: " << student.name << endl;
    cout << "Student ID: " << student.id << endl;

    return 0;
}

File: data_types_project/rust/records.rs
struct Student {
    name: String,
    id: i32,
}

fn main() {
    let student = Student {
        name: String::from("Bryce"),
        id: 1001,
    };

    println!("Student Name: {}", student.name);
    println!("Student ID: {}", student.id);
}